unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/Catalyst/Catalyst.cs

```diff
@@ -41,15 +50,15 @@ public interface ICatalyst : IDisposable
 
 public class Catalyst : ICatalyst
 {
-    private const float CACHE_TIME = 5 * 60;
+    private const float DEFAULT_CACHE_TIME = 5 * 60;
     private const int MAX_POINTERS_PER_REQUEST = 90;
 
     public string contentUrl => realmContentServerUrl;
 
     private string realmDomain = "https://peer.decentraland.org";
     private string realmContentServerUrl = "https://peer.decentraland.org/content";
```

Comment:
Remember that we have `ContentServerUtils.cs` for this kind of stuff. Maybe we should unify this code with that one somehow.

Reply:
Those url are just placeholder it real value is set on class constructor or/and when user moves to other realm https://github.com/decentraland/unity-renderer/blob/3274c0ab77663f65eeb40c68c0498eb5ab88e9d0/unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/Catalyst/Catalyst.cs#L63-L71
I leave it there cause I though it would be easy to understand what those string are for instead of just leaving them as `null`, but I can change them back to `null` if you like